framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java

```diff
@@ -241,6 +245,73 @@ private void executeQuery(String query) throws IOException, SQLException {
     }
   }
 
+  private void executeLimitZeroQuery(String query) throws IOException, SQLException {
+	BufferedWriter writer = new BufferedWriter(new FileWriter(new File(outputFilename),true));
+    final boolean cancelQuery = rand.nextInt(100) < TestDriver.OPTIONS.cancelPercent;
```

Comment:
Yes, we don't need to cancel the limit 0 query. Added a check to exit early if original test is canceled.